permissions/Camera.permission

```diff
@@ -11,3 +11,7 @@
 dbus-system.talk org.maemo.resource.manager
 dbus-system.broadcast org.maemo.resource.manager=org.maemo.resource.manager.*@/*
 # END systembus-org.maemo.resource.manager
+
+# BEG sessionbus-org.sailfishos.zxing
+dbus-user.talk org.sailfishos.zxing
```

Comment:
No need to listen signal from "org.sailfishos.zxing"?

Reply:
Works fine without it.

Reply:
Ok, and if there are issues later it's easy to add "dbus-user.broadcast" rules for this as well.